src/frontend/Typechecker.ml

```diff
@@ -728,34 +728,69 @@ let verify_assignment_global loc cf block is_global id =
   if (not is_global) || block = cf.current_block then ()
   else Semantic_error.cannot_assign_to_global loc id.name |> error
 
-let mk_assignment_from_indexed_expr assop lhs rhs =
-  Assignment
-    {assign_lhs= Ast.lvalue_of_expr lhs; assign_op= assop; assign_rhs= rhs}
-
 let check_assignment_operator loc assop lhs rhs =
```

Comment:
This should now be named `verify` since it doesn't return anything?

Reply:
Good catch, I forgot there was a naming convention.